custom_fsync/sync.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+)
+
+func main() {
+	if len(os.Args) != 1 {
+		fmt.Fprintf(os.Stderr, "Usage: %s\n", os.Args[0])
+		os.Exit(1)
+	}
+
+	err := filepath.WalkDir(".", func(path string, di fs.DirEntry, err error) error {
```

Comment:
WalkDir doesn't seem to follow links. Is that ok? seems like we would want to follow links maybe? idk honestly